media_player.py

```diff
@@ -34,7 +36,7 @@
 DEFAULT_TIMEOUT = 5
 BASE_URL = 'http://{0}/YamahaRemoteControl/ctrl'
 
-ENABLE_OUTPUT_SCHEMA = MEDIA_PLAYER_SCHEMA.extend({
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
```

Comment:
It doesn't look like this change is required, and the bundled components still do things the other way. What benefit does this give?

Reply:
See comment: https://github.com/vrih/homeassistant-rn301/pull/3#discussion_r259585627